plugins/star-rating/api/api.js

```diff
@@ -10,30 +10,349 @@ var plugin = {},
      */
     plugins.internalEvents.push('[CLY]_star_rating');
     plugins.internalDrillEvents.push("[CLY]_star_rating");
+    
+    var createFeedbackWidget = function(ob) {
+        var params = ob.params;
+        var popupHeaderText = params.qstring.popup_header_text;
+        var popupCommentCallout = params.qstring.popup_comment_callout;
+        var popupEmailCallout = params.qstring.popup_email_callout;
+        var popupButtonCallout = params.qstring.popup_button_callout;
+        var popupThanksMessage = params.qstring.popup_thanks_message;
+        var triggerPosition = params.qstring.trigger_position;
+        var triggerBgColor = params.qstring.trigger_bg_color;
+        var triggerFontColor = params.qstring.trigger_font_color;
+        var triggerButtonText = params.qstring.trigger_button_text;
+        var targetDevices = params.qstring.target_devices;
+        var targetPage = params.qstring.target_page;
+        var targetPages = params.qstring.target_pages || [];
+        var isActive = params.qstring.is_active;
+        var hideSticker = params.qstring.hide_sticker;
+        var app = params.qstring.app_key;
+        var collectionName = "web_feedback_widgets_" + app;
```

Comment:
Possible to use app_id instead of app_key for collection name? We user app_id everywhere